src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -83,7 +84,7 @@ public HttpServer vertxServer(Vertx vertx, Configuration conf, Router router)
 
     @Provides
     @Singleton
-    public Router vertxRouter(Vertx vertx, HealthService healthService)
+    public Router vertxRouter(Vertx vertx, HealthService healthService, InfoService infoService)
```

Comment:
Capturing this from jira:

> When start sidecar would be nice to log something indicating how to get to the docs endpoint.

Would still be nice to have this as isn't clear how/where to hit the swagger endpoint and would be a nice UX thing  to log a URL for first time users.

Reply:
On another note I wonder if we should get rid of `docs/index.html` and promote swagger up to docs, as it seems more useful than redoc and includes the examples.